### DISCUSSION TITLE
Sound Change?

### DISCUSSION THREAD

**Discussion by scotthibbs** [w7sst/MorseRunner#99] [Q&A] [answered]
Has the sound changed from the original program from Alex? Things like the CW weight or stations sounding weaker?

**Comment by w7sst** [comment]
@scotthibbs , Not to my knowledge. It's possible that the difference is due to compilation or compilers. It's possible that the code used in generating the CW was compiled in debug-mode (not optimized) and the rest of the program was compiled in release mode (with optimization). I have not seen any changes from Alex's original code in this area, but you never know.

Also, be sure to add the link to the YouTube review where this observation was first observed (I think that is where it came from). Perhaps the author of the YouTube review is available to help us. Can you reach out and have him install 1.68 and have him do a comparison for us? It would be good to get a side-by-side comparison with similar settings. If necessary, I can recompile 1.81 in full release mode so we can eliminate the possibility of the Morse generating engine being compiled in debug mode.

We should perhaps change this discussion into an Issue and see if someone will investigate this. we can mark it with a `help wanted` label. Scott, in the mean time please reach out and gather whatever information you can and include it here. Once all data is collected, we can convert this into an Issue (I think all comments will transfer to the Issue).

Thanks, Mike

**Comment by tekenny** [comment]
This might be unrelated but I’ve wondered if we can do a PARIS test to confirm that WPM are correct…

**Comment by scotthibbs** [comment, marked as answer]
Sound doesn't change while running, this is set at start. 

Since writing the question, I've duplicated the issue on my laptop only. 1.68 plays morse louder than 1.81. My desktop plays them both equally. I'm confused as to why. 

When I get a chance I'll make a video to share amongst us of the issue on both pc and laptop and get an inventory of my sound cards/pc specs to see if this gives a clue on the difference. I'll try other computers as well. (I got a computer or laptop in almost every room here)

I need a little bit to get this done and then I'll create the issue. I don't think it's a right now action item.

**Comment by w7sst** [comment]
Knowing that 1.68 plays louder than 1.81 indicates we have a difference. We
have a different compiler, possible mix of debug and release compiled code,
and the possibility that some code has changed since 1.68. We will have to
do an audit to be sure. There is an explanation. We might have to add some
debug code to both and compare.

**Comment by scotthibbs** [comment]
Just got an email from Jay W4ZXT, he stated this in his video too but I missed it as I concentrated on the loudness. He states in his email, "1.81 seems to have deeper Qsb and the Cw weight sounds about 2.5 instead of the regular most common 3 to one." 

Since he could hear the sound difference, I'm not betting against his ears on this either. :)  So when we make this issue about the sound we will need to add this (or another issue) about the weight. Just wanted to note this for later.

**Comment by stianeklund** [comment]
There is quite a bit of documentation in this issue that highlights some of the differences: https://github.com/w7sst/MorseRunner/issues/301 perhaps helpful to someone else that stumble across this discussion in the future.
